src/stories/components/Popover.stories.mdx

```diff
@@ -100,21 +100,19 @@ export const docsDecorator = () => {
 
 # Popover
 
-### Popovers são pop-ups que surgem quando o usuário clica em um determinado elemento.
-### São usados para emular o comportamento de tooltips, e para mostrar conteúdos que estavam previamente ocultos.
+### O popover é um componente utilizado para mostrar informações contextuais a partir de um clique em determinado elemento. Após o clique a informação oculta do elemento surge por meio de um contêiner, fornecendo informações e ações para aquele contexto especifico.
+### <strong> Observação: </strong> Mantenha o conteúdo do popover breve. Os usuários precisam fazer uma seleção ou inserir informações rapidamente para concluir a tarefa.
 ---
 <br />
 
 ## Recomendamos o uso de Popovers quando:
-- Se deseja colocar filtros ou textos que serão apresentados apenas após a iteração do usuário com algum elemento da interface.
+- For necessário exibir informações contextuais em um elemento junto com uma ação, sem necessitar transitar entre as páginas ou bloquear a página atual.
```

Comment:
Aqui acho que poderia ser "For necessário exibir informações adicionais e ações contextuais sem que seja necessário transitar entre páginas ou remover o foco do usuário da página."

Reply:
@lucasn4s pega essa recomendação